### DISCUSSION TITLE
Nuxt with Bun JS Runtime

### DISCUSSION THREAD

**Discussion by itsmnthn** [nuxt/nuxt#17302] [General]
Has anyone heard about [Bun](https://bun.sh/)? I used it as package manager. It works buttery smooth and faster than `yarn`. I don't have familiar with Nuxi How it works but there should be experimental support after Nuxt3 Releases.

**Comment by amiria703** [comment]
How did you use it as a package manager?
In their homepage, they introduced it as an alternative to nodejs.

**Reply by itsmnthn** [reply]
Yes, it is alternative to Node JS but Bun has built in support for package managing and it is compatible with npm packages so we can do `bun install vue` and it works fine and we can use node js to run `vue` project as we normally we would do. It can be used as package manager. but it would be great if we can run nuxt app with Bun then it would be hell of speed since we don't need ts to js transpiling.

**Comment by theguriev** [comment]
I tried bun but didn't notice much difference in performance. Is there any evidence that it is actually faster?
